code/modules/overmap/ships/ship.dm

```diff
@@ -100,6 +104,13 @@ var/const/OVERMAP_SPEED_CONSTANT = (1 SECOND)
 /obj/effect/overmap/visitable/ship/proc/adjust_speed(n_x, n_y)
 	CHANGE_SPEED_BY(speed[1], n_x)
 	CHANGE_SPEED_BY(speed[2], n_y)
+	var/magnitude = norm(n_x, n_y)
+	var/inertia_dir = magnitude >= 0 ? turn(fore_dir, 180) : fore_dir
+	var/inertia_strength = magnitude * 1e3
+	if(needs_dampers && damping_strength < inertia_strength)
+		for(var/area/A in world)
```

Comment:
`*scream`
Isn't there an area list somewhere?

Reply:
`area_repository` has a bunch of helpers for quickly finding areas matching criteria.